tests/ert_tests/ert3/console/integration/test_cli.py

```diff
@@ -666,6 +681,75 @@ def test_cli_local_test_run_specific_realization(tmpdir):
             assert pathlib.Path(f"local-test-run-{run_id}/output.json").exists()
 
 
+def test_cli_local_test_run_storageinput(tmpdir):
+    with chdir(tmpdir):
+        with Storage.start_server():
+            args = ["ert3", "init", "--example", "polynomial"]
+            with patch.object(sys, "argv", args):
+                ert3.console._console._main()
+
+            os.chdir("polynomial")
+
+            for tainted_repo_dir in glob.glob("local-test-run*"):
+                # Because the runpath for local test runs is unpredictable,
+                # we need this in order to reliably guess where to look.
+                shutil.rmtree(tainted_repo_dir)
```

Comment:
Why do cleanup here?

Reply:
This is mostly for the developers convenience (letting you run `pytest` without `git clean` upfront in some cases), but also serves to demonstrate the weak point in the code on unpredictable runpaths.